libmscore/spanner.cpp

```diff
@@ -605,7 +605,7 @@ void Spanner::computeEndElement()
                         Fraction tick = (l->ticks().ticks() == Lyrics::TEMP_MELISMA_TICKS) ? l->tick() : l->endTick();
                         Segment* s = score()->tick2segment(tick, true, SegmentType::ChordRest);
                         if (!s) {
-                              qDebug("%s no end segment for tick %d", name(), tick);
+                              qDebug("%s no end segment for tick %d/%d", name(), tick.numerator(), tick.denominator());
```

Comment:
or use `qDebug("%s no end segment for tick %d", name(), tick.ticks();` instead?

Reply:
I actually changed this to tick.ticks() in another or yesterday, so if you do the same, that would lessen the chance of a merge conflict I assume.

Reply:
done, seen it being used elsewhere

Reply:
Done the same way in #4730